addons/iptables.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
But I'll merge this, and then that refactoring can take place in the mantl project whenever it's ready!